web/packages/design/src/SlideTabs/SlideTabs.tsx

```diff
@@ -69,7 +73,7 @@ type props = {
   // The style to render the selector in.
   appearance?: 'square' | 'round';
   // The index that you'd like to select on the initial render.
-  initialSelected?: number;
+  activeIndex: number;
```

Comment:
Good change. 👍 I mean controlling the current tab from outside of the component.

Could you update the comment and turn all comments into JSDocs?